### DISCUSSION TITLE
Main Board v4.3.1 Display boad v1.4

### DISCUSSION THREAD

**Discussion by canadian062474** [Jyers/Marlin#2007] [Q&A]
I just bought  an Ender 3 v2  I also installed  the CR touch  i am look to flash the firmware but i don't see on that will work with my version . IF any one can help me out that would be great. Thanks

**Comment by carioca53** [comment]
Hi Canadian, where are you located I may be able to help you

**Comment by canadian062474** [comment]
Ontario  canada

**Reply by carioca53** [reply]
I am in Uxbridge, ON

**Reply by canadian062474** [reply]
St Mary's Here. This is the mother board. i use to updating firmware and i Know if i install the wrong update it will brick the unit 
![20220811_140133](https://user-images.githubusercontent.com/111021073/184210254-f72f4f4e-ff0f-4def-abf4-ff638ed8f278.jpg)
e

**Reply by carioca53** [reply]
It appears the 4.3.1 board is comparable to a 4.2.2 board on the Ender 3V2. You should be able to use 4.2.2 version firmware on this board without an issue. the 4.3.1 board isn't a good board, it's a lesser version of the  4.2.2 board so I would recommend getting a 4.2.7 board, you will have more options for firmware with the 4.2.7 board. There is a version of Jyers from Jan 21/22 that should work but you have to compile your own firmware, I can compile a copy for you but would rather do it via emails.

**Comment by canadian062474** [comment]
Thank you for the help. I was able to find a firmware that worked. Mriscoc
Professional Firmware. I will update the board more then likely soon.
Thank you again for the quick response and help.   I found the answer also
on  credulity ender 3 v2 Facebook page.

**Reply by carioca53** [reply]
Good choice with Mriscoc pro firmware, very reliable, I loaded in my grandson E3V2, works perfectly with lots of options.

**Comment by canadian062474** [comment]
I'm just running my first print right now. Since my wife said I can buy it
I'm making the owl for her . Happy wife is a happy hubby.

**Reply by carioca53** [reply]
after 43 years I can attest to happy wife is a happy hubby.
